### DISCUSSION TITLE
Still confusing

### DISCUSSION THREAD

**Discussion by raselldev** [dotnet/maui#20800] [Q&A]
I still confuse with Microsoft.  They create cross platform framework but, in internal development like Ms Teams and XBox, they using React Native.

**Comment by ghost** [comment]
[This is a good read for a start](https://devclass.com/2024/04/11/react-native-and-why-microsoft-uses-it-for-its-own-cross-platform-development/) @raselldev , this video could be [interesting](https://devblogs.microsoft.com/react-native/2022-09-02-rneu-microsoft-billions/) (CoPilot Ai found those links).
